package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "npm run build:icon-index && npm run build:babel && npm run build:copy-files",
     "build:icon-index": "babel-node ./scripts/icon-index-generator.js",
-    "build:babel": "babel ./src --ignore *.spec.js --out-dir ./build",
+    "build:babel": "node --stack-size=10000 ./node_modules/.bin/babel ./src --ignore *.spec.js --out-dir ./build",
```

Comment:
Are you sure this works on Windows? Shouldnt it be more like `node --stack-size=10000 ./node_modules/babel-cli/bin/babel.js ./src --ignore *.spec.js --out-dir ./build`?

Because .bin/babel is a linux file